src/zjs_modules.c

```diff
@@ -95,16 +96,26 @@ static jerry_value_t native_require_handler(const jerry_value_t function_obj,
     for (int i = 0; i < modcount; i++) {
         module_t *mod = &zjs_modules_array[i];
         if (!strcmp(mod->name, module)) {
-            return jerry_acquire_value(mod->init());
+            // We only want one intance of each module at a time
+            if (mod->instance == 0) {
+                mod->instance = jerry_acquire_value(mod->init());
+            }
+            return mod->instance;
         }
     }
 
-    PRINT("MODULE: `%s'\n", module);
```

Comment:
This should stay there. It's meant to tell you which module failed to load in the unlikely event of such a failure. (But a typo in require() would do this maybe?)

It used to be part of the print but I pulled it out so zjs_error wouldn't need to take variable arguments.